src/controller/CHIPDeviceController.cpp

```diff
@@ -710,6 +719,8 @@ void DeviceCommissioner::OnRendezvousComplete()
     device = &mActiveDevices[mDeviceBeingPaired];
     mPairedDevices.Insert(device->GetDeviceId());
     mPairedDevicesUpdated = true;
+    mSessionManager->NewPairing(Optional<Transport::PeerAddress>(mRendezvousSession->GetPeerAddress()), device->GetDeviceId(),
+                                &mRendezvousSession->GetPairingSession());
```

Comment:
Why this change ?

Reply:
Removed.

However, to maintain the secure session inside Deivce is a bit weird for me. I agree that we need to persist device session keys and restore it later.

IMHO, this piece of code should not exist in any part of the device controller (as well as the example apps) and the pairing key should be inserted to the secure session manager by rendezvous session or secure pairing session directly. The the work for persistent the keys should also be done by secure session manager.

This might be a bit out of topic of what this PR does. I just remove this line to be consistent with current implementation, we can discuess about this later.

Reply:
After some attempts, I added this line back, with the comment that the work for adding keys to securesession manager should be done by rendezvous session or pairing session.

Reply:
After some more attempts, CommandSender() in ChipDevice no longer listens to pairing state.

That makes sense for me since CommandSender is placed after the MessageLayer and should not care about the lower pairing state. Even if the lower connection is lost, message layer can handle it, so I added a InitCommandSender() in ChipDevice to init the command sender when we have a valid pairing.

Reply:
Please keep some similar logic to `ChipDevice::SendMessage` into `ChipDevice::SendCommands` when it comes to play with keys.
Then, fill a followup to change the logic.
But maybe I'm wrong and this is totally fine, it's been a few months since I played with the code under `src/transport`.  @pan-apple do you have any opinions ?

Reply:
I'm not sure what this has been marked as resolved. I would like @pan-apple to look at this change. I do not fully understand why we need to call `NewPairing` here.